packages/hidp_wagtail/pyproject.toml

```
@@ -12,7 +12,7 @@ authors = [
     { name = "Wouter de Vries", email = "wdevries@leukeleu.nl" },
 ]
 dependencies = [
-    "django-hidp>=1.4.0,<1.5.0",
+    "django-hidp>=1.5.0,<2.0",
```

```suggestion
    "django-hidp>=1.5.0,<2",
```
Is a bit cleaner, and more conventional.